tests/test_jobs.py

```diff
@@ -131,7 +131,7 @@ def test_job_start_override_hardware_tier_id(default_domino_client):
     )
     assert job_status["statuses"]["isCompleted"] is True
     job_red = default_domino_client.job_runtime_execution_details(job_status["id"])
-    assert job_red["hardwareTierId"] == override_hardware_tier_id
+    assert job_red["hardwareTier"]['id'] == override_hardware_tier_id
```

Comment:
Nit: There are a few cases in this PR where you have a weird mix of `"` and `'`, might be nice to pick one for consistency